Sources/WordPressUI/Extensions/UIImageView+Gravatar.swift

```diff
@@ -2,9 +2,23 @@ import Foundation
 import UIKit
 import Gravatar
 import enum Gravatar.Rating
+@_exported import struct Gravatar.AvatarURL
 
 public typealias ImageRating = Rating
 
+@available(*, deprecated, renamed: "AvatarURL")
+public typealias GravatarURL = AvatarURL
+
+extension GravatarURL {
+    public init?(_ url: URL) {
+        self.init(url: url)
+    }
+
+    public var canonicalURL: URL {
+        self.canonicalUrl
```

Comment:
I wish we also named it as "canonicalURL", wouldn't it be better?

Reply:
Yup, let's do that 👍